applications/nvidia-gpu-operator/25.3.2/defaults/cm.yaml

```diff
@@ -36,13 +36,13 @@ data:
         enabled: true
         additionalLabels:
           prometheus.kommander.d2iq.io/select: "true"
-      version: 4.2.3-4.1.3-ubuntu22.04
+      version: 4.2.3-4.2.0-ubuntu22.04
     validator:
       repository: nvcr.io/nvidia/cloud-native
-      version: v25.3.1
+      version: v25.3.2
     operator:
       repository: nvcr.io/nvidia
-      version: v25.3.1
+      version: v25.3.2
     migManager:
```

Comment:
release notes say that mig manager was bumped to 12.2 - `NVIDIA MIG Manager for Kubernetes v0.12.2`
https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/release-notes.html

Can you please verify other components - https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/platform-support.html#gpu-operator-component-matrix